cmd/plakar/subcommands/create/plakar-create.1

```diff
@@ -19,6 +19,10 @@ which defaults to
 .Pp
 The options are as follows:
 .Bl -tag -width Ds
+.It Fl hashing Ar algorithm
+Provide alternative hashing algorithm to replace the default.
+Supported algorithms are BLAKE3 and SHA256, default is BLAKE3.
+If specified, the repository will not use encryption.
```

Comment:
copy-paste? :p 

while there, can you bump the date in the manpage as well? (on cvs it's magic, on git has to be done by hand)

Reply:
eeek yeah copy paste as a template, let me fix